api-gateway-routes/README.md

```diff
@@ -3,3 +3,9 @@
 This lambda function defines the actions that happen when the Javabuilder API Gateway triggers the connect, disconnect, and default actions.
 
 To create a zip file to upload to the lambda, run [zip_for_aws](zip_for_aws.sh).
+
+# Prerequisites
+You need to install Ruby 2.7.4 to build and/or run this function. [rbenv](https://github.com/rbenv/rbenv) is a useful tool for installing Ruby versions
```

Comment:
@jmkulwik mind checking if these instructions are ok for a windows setup?

Reply:
It may be correct, but `zip_for-aws.sh` doesn't work on a windows setup - I just use my EC2 instance right now. For both of these, someone would likely need WSL.

Reply:
Should we remove the Read Me files for each Lambda and use a consolidated Read Me at the root of the project? The deploy shell script at the root of the project builds and deploys the 2 Ruby Lambdas (and the Java Lambda). We probably don’t need the `.zip_for_aws.sh` scripts inside each Lambda directory anymore either.

Reply:
I've moved this text to the main readme and deleted the `.zip_for_aws.sh` scripts.